src/reducers/updateColumnWidth.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
same as updateRowHeight

except we don't have an equivalent to 'storedHeights'